plugins.json

```diff
@@ -30,6 +30,16 @@
 			"entrypoint": "HitboxPlugin.cpp",
 			"dllname": "hitboxplugin.dll",
 			"settings": "lib/hitboxplugin.set"
+		},
+		{
+			"name": "CurveballPlugin",
+			"author": "https://github.com/Aberinkula",
+			"repo": "https://github.com/Aberinkula/BendItLikeBeckham",
+			"submodule": "BendItLikeBeckham",
+			"sourcedir": "",
+			"entrypoint": "BendItPlugin.cpp",
+			"dllname": "Release/benditplugin.dll",
```

Comment:
I should probably make this more clear, but the dllname is the name it will be compiled to (in this repository), so it's not the path to your already compiled DLL. Let me see if I can fix it in this PR somehow

Reply:
Eh, I'll just merge this then do it manually. Much easier that way